addons/bus/static/tests/bus_tests_tour.js

```diff
@@ -0,0 +1,23 @@
+odoo.define("bus.tour", function(require) {
+"use strict";
+
+const tour = require("web_tour.tour");
+
+tour.register("bundle_changed_notification", {
+    test: true,
+}, [{
+        trigger: '.o_notification_title:contains(Update available)',
+    },
+    tour.stepUtils.showAppsMenuItem(),
+    {
+        trigger: '.o_app[data-menu-xmlid="base.menu_administration"]',
+        position: "bottom",
+    }, {
+        extra_trigger: '.o_menu_brand:contains(Settings)',
+        trigger: '.o_notification_buttons button:contains(Refresh)',
+    }, {
+        trigger: '.o_menu_brand:contains(Settings)',
```

Comment:
This doesn't actually check that it reloads, right? (but I don't know how to do that). My point being, this step is maybe useless.